### PR TITLE
Adds suppressing of URI labels and applies to base templates.

### PR DESCRIPTION
**Pull request by justinlittman**
refs #3154

## Why was this change made?
Less confusing templates.

## How was this change tested?
image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -310,6 +310,7 @@ The following are only in the resource subject (that is, the base subject).
   repeatable: <true | false>
   ordered: <true | false>
   languageSuppressed: <true | false>
+  labelSuppressed: <true | false>
```
Comment (justinlittman) [inline README.md:313]: This is the new property template attribute in the state model.

File: __tests__/__action_fixtures__/subjectTemplate-suppressible.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const subjectTemplate = build.subjectTemplate({
       uris: {
         "http://sinopia.io/testing/Suppressible/property1": "Property1",
       },
-      valueSubjectTemplateKeys: ["resourceTemplate:testing:uri"],
+      valueSubjectTemplateKeys: ["resourceTemplate:testing:suppressedUri"],
```
Comment (justinlittman) [inline __tests__/__action_fixtures__/subjectTemplate-suppressible.js:21]: I switched around two template fixtures, so a handful of changes like this where they were used.

File: src/TemplatesBuilder.js [hunk 3]
```diff
@@ -169,6 +169,11 @@ export default class TemplatesBuilder {
     )
     if (attributeTerm) {
       propertyTemplate.defaults = this.defaultsForUri(attributeTerm)
+      const uriAttrValues = this.valuesFor(
```
Comment (justinlittman) [inline src/TemplatesBuilder.js:172]: This is where suppressed label gets loaded in the TemplateBuilder.

File: src/actionCreators/resourceHelpers.js [hunk 4]
```diff
@@ -504,7 +504,7 @@ const newUriFromObject = (obj, property, propertyUri, context) => {
   const labelQuads = context.dataset
     .match(obj, rdf.namedNode("http://www.w3.org/2000/01/rdf-schema#label"))
     .toArray()
-  let label = uri
+  let label = null
```
Comment (justinlittman) [inline src/actionCreators/resourceHelpers.js:507]: No longer defaulting the label for a URI to that URI when a label is not present.

**Review by mjgiarlo — APPROVED** [review]

**Pull request merged**